### ISSUE TITLE
Classic Era version broken

### ISSUE THREAD

**Issue by GeorgeChatzigiannis** [Lethay/TalentedClassic#108]
Following issues found:

- Template apply no longer applies talents
- Cannot create new template with custom name

Looks like overall the addon is not functioning properly at all.

**Comment by mimixy** [comment]
Same issue here. No LUA error. Pressing Apply Template just doesn't do anything. I waited 30 mins, still nothing.

**Comment by Ulidan** [comment]
Created a fork with a fixed Era version.
Hope Lethay will get to it.

**Comment by GeorgeChatzigiannis** [comment]
@Ulidan I tried your fork but it looks like after logging out from the client the templates are not saved. I haven't tried yet applying any template yet.

**Comment by GeorgeChatzigiannis** [comment]
I'll do some further testing but for now I noticed Actions>Options throws this error and is not working:

`Interface/AddOns/Talented/ui/menu.lua:27: Talented isn't registed with AceConfigRegistry, unable to open config`

[string "@Interface/AddOns/Talented/ui/menu.lua"]:27: in function `OpenOptionsFrame
[string "@Interface/AddOns/Talented/ui/menu.lua"]:351: in function `func`


cc @Ulidan

**Comment by TheCyberKnight** [comment]
I can tall you more of what is happening. I used talented as a rogue to put 5 points in Subtlety tree into Master of deception. then 2 points into Camouflage, when I went to stick a 3rd point into Camo it would not work. I figured out kinda the reason why. after turning the addon off I realized Talented shows two points into Camo but with addon off those two points are not in camo but in Elusiveness to the left of camo, and Elusive can only have two points max. So for some reason the skill icons on talented are not putting points into the correct skills. Unfortunatly this is my first character and now I am specced wrong and have to respec. I do not even have 1 gold yet. I would recommend not using this addon untill fixed. for those of you not able to spend points log in with addon off and see if your specced different than you think. Even if you do not see anything weird, the point your trying to spend that is not working is probably trying to go to the wrong skill anyways.
